main.go

```diff
@@ -282,12 +508,7 @@ func (c *CASProxy) ValidateTicket(w http.ResponseWriter, r *http.Request) {
 	// the CAS server fairly often. Adjust the max age to rate limit requests to
 	// CAS.
 	var s *sessions.Session
-	s, err = c.sessionStore.Get(r, sessionName)
-	if err != nil {
-		err = errors.Wrap(err, "error getting session")
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
+	s, _ = c.sessionStore.Get(r, sessionName)
```

Comment:
Checking the error here was hampering local testing. Since this is very likely to be a new session, I chose simply not to check for an error. Any error that is returned here is most likely caused by an old session that couldn't be decrypted using the session encryption key.